Source/ARTCrypto.h

```diff
@@ -44,11 +51,30 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ The ARTCrypto object ensures that message payloads are encrypted, can never be decrypted by Ably, and can only be decrypted by other clients that share the same secret symmetric key.
+ */
 @interface ARTCrypto : NSObject
 
-+ (ARTCipherParams *)getDefaultParams:(NSDictionary *)values;
+/**
+ Retrieves, or optionally sets, the `ARTCipherParams` for the channel.
+ @param params Overrides the default parameters. A suitable key must be provided as a minimum.
+ @return `ARTCipherParams` A ARTCipherParams object, using the default values for any field not supplied.
+ */
++ (ARTCipherParams *)getDefaultParams:(NSDictionary *)params;
+
+/**
+ Generates a random key to be used in the encryption of the channel.
+ @param keyLength The length of the key, in bits, to be generated. If not specified, this is equal to the default keyLength of the default algorithm: for AES this is 256 bits.
```

Comment:
```suggestion
 @param keyLength The length of the key, in bits, to be generated. If not specified, this is equal to the default key length of the default algorithm: for AES this is 256 bits.
```

@tbedford @m-hulbert I think an encryption algorithm is an abstract concept and not a class in our SDK, so I'm not sure that using `keyLength`, which suggests a property name, is appropriate here. Not sure though.

Reply:
@QuintinWillison are you able to confirm this if possible?